oryx-app-mllib/src/test/java/com/cloudera/oryx/app/mllib/kmeans/KMeansEvalIT.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright (c) 2015, Cloudera and Intel, Inc. All Rights Reserved.
+ *
+ * Cloudera, Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"). You may not use this file except in
+ * compliance with the License. You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ * CONDITIONS OF ANY KIND, either express or implied. See the License for
+ * the specific language governing permissions and limitations under the
+ * License.
+ */
+
+package com.cloudera.oryx.app.mllib.kmeans;
+
+import java.util.List;
+
+import org.apache.spark.api.java.JavaRDD;
+import org.apache.spark.mllib.linalg.Vector;
+import org.junit.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.cloudera.oryx.app.kmeans.ClusterInfo;
+
+public final class KMeansEvalIT extends AbstractKMeansEvalIT {
```

Comment:
OK, but you are still duplicating the tests. I'm not against it, but it seems redundant. I don't see the need for a second abstract superclass here. There's just one subclass, that I thought might only test things like `testFetchSampleEvalData`. I don't see a test of the sampling?